Source/DafnyLanguageServer/Handlers/DafnyHoverHandler.cs

```diff
@@ -132,7 +135,7 @@ class AssertionBatchIndexComparer : IComparer<AssertionBatchIndex> {
         }
 
         information += "  \n";
-        if (assertionBatchesToReport.Count == 1) {
+        if (orderedAssertionBatches.Count == 1) {
```

Comment:
Is this change actually needed? (This is mostly curiosity, I think it's correct in any case)

Reply:
This change prevents `if (assertionBatchesToReport.Count == 1) { ...; orderedAssertionBatches.First()` }` from throwing a "Sequence has no elements" exception, does it not?